sci-libs/libccd/libccd-2.1.ebuild

```diff
@@ -19,14 +19,10 @@ HOMEPAGE="http://libccd.danfis.cz/
 
 LICENSE="BSD"
 SLOT="0"
-IUSE="+double-precision doc +shared test"
```

Comment:
...this is even worse that `static-libs`. The crap gentoo devs really do :unamused: